data/genesis/imap/bug/duplicatefolder.rb

```diff
@@ -55,7 +55,7 @@
   #Normal operation
   proxy(mimap.method('create'),"blurdybloop"), 
   RenameVerify.new(mimap, "blurdybloop","Trash/bar"),   
-  v(RunCommand.new('/opt/zimbra/bin/mysql', Command::ZIMBRAUSER, '-X -e "select id,group_id,comment from zimbra.mailbox;"')) do |mcaller, data|
+  v(RunCommandOnMailbox.new('/opt/zimbra/bin/mysql', Command::ZIMBRAUSER, '-X -e "select id,group_id,comment from zimbra.mailbox;"')) do |mcaller, data|
```

Comment:
This should be _safe_ to do, since in an HA environment, all mailboxes will share the same database instance.  The only hassle/consideration is that we will need to continue to install/configure/run STAF on the mailbox nodes.